sympy/physics/control/lti.py

```diff
@@ -1100,7 +1103,115 @@ def is_stable(self):
         True
 
         """
-        return fuzzy_and(pole.as_real_imag()[0].is_negative for pole in self.poles())
+        try:
+            standard_form = self.to_standard_form()
+            s = standard_form.get_asymptotic_stability_conditions()
+            output = reduce_inequalities(s)
+            if output in (true, false):
+                return output
+
+            return None
+        except NotImplementedError:
+            # If there are more than one symbols,
+            # reduce_inequalities could fail
+            return None
```

Comment:
try/except should be used around the smallest part of the code possible. If this is catching an exception from `reduce_inequalities` then only that call should be in the `try` block.